lib/index.js

```diff
@@ -55,6 +55,7 @@ internals.schema = Joi.object({
     ttl: Joi.number(),
     domain: Joi.string().allow(null),
     providerParams: Joi.object(),
+    allowRuntimeQueryParams: Joi.boolean().default(false),
```

Comment:
Renaming it to `allowRuntimeProviderParams` might make the documentation easier since they are related.